eventstore/model.go

```diff
@@ -284,6 +303,14 @@ func (t *Txn) FindByID(id core.EntityID, v interface{}) error {
 	if err != nil {
 		return err
 	}
+	if t.model.jsonMode {
+		str := string(bytes)
+		rflStr := reflect.ValueOf(str)
+		reflV := reflect.ValueOf(v)
+		reflV.Elem().Set(rflStr)
+
+		return nil
+	}
```

Comment:
In `FindByID` we'll store in `v` (which should be a `*string`) just the plain byte json we got from the store. In this mode we avoid the `json.Unmarshal` to return a Go Type.